commit 6c3abd83758060356db4fa4e9d69d5bec09865e4
Author: Jeff Layton <jlayton@redhat.com>
Date:   Mon Jan 4 15:42:51 2010 -0500

    mount.nfs: don't use IPv6 unless IPV6_SUPPORTED is set
    Commit 1f3fae1fb25168aac187ff1881738c8ad53a8763 made mount.nfs start
    looking up and trying to use IPv6 addresses when mount.nfs was built
    against libtirpc (even when --enable-ipv6 wasn't specified).
    
    The problem seems to be that nfs_nfs_proto_family() is basing the family
    on HAVE_LIBTIRPC. I think it should be basing it on IPV6_SUPPORTED
    instead.
    
    Signed-off-by: Jeff Layton <jlayton@redhat.com>
    Acked-by: Chuck Lever <chuck.lever@oracle.com>
    Signed-off-by: Steve Dickson <steved@redhat.com>

diff --git a/utils/mount/network.c b/utils/mount/network.c
index e3ad5c2..906e20c 100644
--- a/utils/mount/network.c
+++ b/utils/mount/network.c
@@ -1344,7 +1344,7 @@ int nfs_nfs_proto_family(struct mount_options *options,
 	unsigned long protocol;
 	char *option;
 
-#ifdef HAVE_LIBTIRPC
+#ifdef IPV6_SUPPORTED
 	*family = AF_UNSPEC;
 #else
 	*family = AF_INET;
